lib/pagseguro/subscription_change_payment/request_serializer.rb

```diff
@@ -0,0 +1,80 @@
+module PagSeguro
+  class SubscriptionChangePayment
+    class RequestSerializer
+      extend Forwardable
+
+      def_delegators :object, :subscription_payment_method
+      def_delegators :subscription_payment_method, :holder
+      def_delegators :holder, :billing_address
+
+      attr_reader :object
+
+      def initialize(object)
+        @object = object
+      end
+
+      def serialize
+        build.to_xml(save_with:
+          Nokogiri::XML::Node::SaveOptions::NO_EMPTY_TAGS |
+          Nokogiri::XML::Node::SaveOptions::FORMAT
+        )
+      end
+
+      private
+
+      def build
+        Nokogiri::XML::Builder.new(encoding: PagSeguro.encoding) do |xml|
+          xml.send(:paymentMethod) {
+            xml.send(:type, 'CREDITCARD')
+
+            if object.sender
+              xml.send(:sender) {
+                xml.send(:hash_, object.sender.hash)
+                xml.send(:ip, object.sender.ip)
+              }
+            end
+
+            if object.subscription_payment_method
```

Comment:
Using the method **serialize_sender** to serialize the sender attributes...
**serialize_subscription_payment_method** to serialize the subscription_payment_method attributes...